slick.dataview.js

```diff
@@ -383,9 +383,20 @@
       return rows.length;
     }
 
+    // (groups: Object[]) => Object[]
+    function getVisibleFlattenedGroups(groups) {
+      var flattenedGroups = [].concat(groups)
+      groups.forEach(group => {
```

Comment:
slickgrid is generally compatible with old browsers, not sure we want to break that compatability here.